template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/Areas/IdentityService/Controllers/AccountController.cs

```diff
@@ -0,0 +1,419 @@
+﻿using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.Service;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.Extensions.Logging;
+using Company.WebApplication1.Identity.Models;
+using Company.WebApplication1.Identity.Models.AccountViewModels;
+using Company.WebApplication1.Identity.Services;
+
+namespace Company.WebApplication1.Identity.Controllers
+{
+    [Area(IdentityServiceConstants.Tenant)]
+    [Route(IdentityServiceConstants.RoutePrefix + "/[area]/[controller]/[action]")]
+    [Authorize(IdentityServiceOptions.LoginPolicyName), AllowAnonymous]
```

Comment:
I'm just curious why the change from using a Constant to using an inline string? Seems to me, using a constant would be a better Practice, since this could be easily updated in one file, and update all of the Controllers within the area...